controllers/clusterresources/helpers.go

```diff
@@ -76,6 +76,4 @@ func getUserCreds(secret *k8sCore.Secret) (username, password string, err error)
 	return username[:len(username)-1], password[:len(password)-1], nil
 }
 
-func getDeletionUserFinalizer(clusterID string) string {
-	return models.DeletionUserFinalizer + clusterID
-}
+const msgDeleteUser = "If you want to delete the user, first remove the user from cluster specification"
```

Comment:
Seems better to say:
```suggestion
const msgDeleteUser = "If you want to delete the user, first remove it from the clusters specifications"
```

Reply:
changed